### PR TITLE
UI implementation to Support separate client side MTLS for production and sandbox endpoints

### PR DESCRIPTION
**Pull request by RusJaI**
Fixes : https://github.com/wso2/api-manager/issues/2433

### CODE REVIEW

**Review by lasanthaS — APPROVED** [review]

File: portals/publisher/src/main/webapp/source/src/app/components/Apis/Details/Endpoints/GeneralConfiguration/UploadCertificate.jsx [hunk 1]
```diff
@@ -296,7 +386,7 @@ export default function UploadCertificate(props) {
                             margin='normal'
                             variant='outlined'
                             error={
-                                (aliasValidity && !aliasValidity.isValid) || (aliasList && aliasList.includes(alias))
+                                (aliasValidity && !aliasValidity.isValid) ||isAliasIncluded()
```
Comment (lasanthaS) [inline portals/publisher/src/main/webapp/source/src/app/components/Apis/Details/Endpoints/GeneralConfiguration/UploadCertificate.jsx:389]: add space after `||`

**Review by lasanthaS — CHANGES_REQUESTED** [review]

File: portals/publisher/src/main/webapp/source/src/app/components/Apis/Details/Endpoints/GeneralConfiguration/UploadCertificate.jsx (hunk 1 above)
Comment (RusJaI) [reply portals/publisher/src/main/webapp/source/src/app/components/Apis/Details/Endpoints/GeneralConfiguration/UploadCertificate.jsx]: fixed

**Review by lasanthaS — DISMISSED** [review]

**Review by Krishanx92 — APPROVED** [review]

File: portals/publisher/src/main/webapp/source/src/app/components/Apis/Details/Endpoints/GeneralConfiguration/UploadCertificate.jsx [hunk 2]
```diff
@@ -137,6 +147,7 @@ export default function UploadCertificate(props) {
     const [isEndpointEmpty, setIsEndpointEmpty] = useState(false);
     const [isPoliciesEmpty, setPoliciesEmpty] = useState(false);
     const [aliasValidity, setAliasValidity] = useState();
+    const [keyType, setkeyType] = useState(API_SECURITY_KEY_TYPE_PRODUCTION);
```
Comment (npamudika) [inline portals/publisher/src/main/webapp/source/src/app/components/Apis/Details/Endpoints/GeneralConfiguration/UploadCertificate.jsx:150]:
```suggestion
    const [keyType, setKeyType] = useState(API_SECURITY_KEY_TYPE_PRODUCTION);
```

Comment (RusJaI) [reply portals/publisher/src/main/webapp/source/src/app/components/Apis/Details/Endpoints/GeneralConfiguration/UploadCertificate.jsx]: fixed

**Review by npamudika — APPROVED** [review]

**Review by npamudika — APPROVED** [review]

**Pull request merged**